staging/jenkins/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+name: jenkins
+home: https://jenkins.io/
+version: 1.7.3
+appVersion: lts
```

Comment:
"lts" seems very broad, dont we want to pin to a more explicit version?

Reply:
:heavy_check_mark: if there's a semver compat release version available let's use that instead.